packages/taco/src/conditions/schemas/rpc.ts

```diff
@@ -20,17 +20,35 @@ const BlockOrContextParamSchema = z.union([
 // eth_getBalance schema specification
 // - Ethereum spec: https://ethereum.github.io/execution-apis/api-documentation/
 // - web3py: https://web3py.readthedocs.io/en/stable/web3.eth.html#web3.eth.Eth.get_balance
-export const rpcConditionSchema = baseConditionSchema.extend({
-  conditionType: z.literal(RpcConditionType).default(RpcConditionType),
-  chain: z.number().int().nonnegative(),
-  method: z.enum(['eth_getBalance']),
-  parameters: z.union([
-    // Spec requires 2 parameters: an address and a block identifier
-    z.tuple([EthAddressOrContextVariableSchema, BlockOrContextParamSchema]),
-    // Block identifier can be omitted, since web3py (which runs on TACo exec layer) defaults to 'latest',
-    z.tuple([EthAddressOrContextVariableSchema]),
-  ]),
-  returnValueTest: returnValueTestSchema, // Update to allow multiple return values after expanding supported methods
-});
+export const rpcConditionSchema = baseConditionSchema
+  .extend({
+    conditionType: z.literal(RpcConditionType).default(RpcConditionType),
+    chain: z.number().int().nonnegative(),
+    method: z
+      .enum(['eth_getBalance'])
+      .describe("Only 'eth_getBalance' method is supported"),
+    parameters: z.union([
+      // Spec requires 2 parameters: an address and a block identifier
+      z
+        .tuple([
+          EthAddressOrContextVariableSchema,
+          BlockOrContextParamSchema.describe(
+            "Block identifier can be omitted, since web3py (which runs on TACo exec layer) defaults to 'latest'",
+          ),
+        ])
+        .describe(
+          'Spec requires 2 parameters: an address and a block identifier',
+        ),
+      z
+        .tuple([EthAddressOrContextVariableSchema])
+        .describe(
+          "Block identifier can be omitted, since web3py (which runs on TACo exec layer) defaults to 'latest'",
+        ),
+    ]),
+    returnValueTest: returnValueTestSchema, // Update to allow multiple return values after expanding supported methods
+  })
+  .describe(
+    'eth_getBalance schema specification\n- Ethereum spec: https://ethereum.github.io/execution-apis/api-documentation/\n- web3py: https://web3py.readthedocs.io/en/stable/web3.eth.html#web3.eth.Eth.get_balance',
```

Comment:
```suggestion
    'RPC Condition for calling Ethereum JSON RPC APIs',
```

Reply:
Done! And I added a link to Ethereum JSON RPC specs.